asmcomp/cmmgen.ml

```diff
@@ -978,8 +978,15 @@ and transl_prim_1 env p arg dbg =
   | Pget_header m ->
       box_int dbg Pnativeint m (get_header (transl env arg) dbg)
   | Pperform ->
-      let cont = make_alloc dbg Obj.cont_tag [int_const dbg 0] in
-      Cop(Capply typ_val,
+      (* We have to hard-code this at the moment since the system compiler's
+         stdlib doesn't have [Obj.cont_tag]. *)
+      let cont_tag = 245 in
+      let cont =
```

Comment:
Please put a fatal error here as well; by the time this becomes relevant, we should at least have access to `runtimetags.ml` (https://github.com/ocaml-flambda/flambda-backend/pull/1857) which would make this safer.

Reply:
Fixed, and added fatal errors in the other cases